wagtail/admin/static_src/wagtailadmin/js/page-editor.js

```diff
@@ -176,7 +176,8 @@ function InlinePanel(opts) {
 
     self.updateAddButtonState = function() {
         if (opts.maxForms) {
-            var forms = $('> li', self.formsUl).not('.deleted');
+            var forms = $('> [data-inline-panel-child]', self.formsUl).not('.deleted');
+            console.log('form count', forms.length);
```

Comment:
rogue `console.log`?

Reply:
Oops! Good catch - removed in 1c7affb1fe2e4aecee5fdf7304bdc946ca104f34.